### PR TITLE
More tooltips

### PR DESCRIPTION
**Pull request by DarksightKellar**
This PR adds a bunch of tooltips across the UI.

Will definitely need copy audits on this as a follow-up.

Also feel free to either suggest removals or additions wherever it catches your eye.

I also 
- fixed a small bug that had rewrap using the sarco's initial resurrection time as the furthest allowable new resurrection time.
- Added the "edit" icon from design (links to details screen from sarco table)
- run prettier, so a couple hooks have formatting diffs

### CODE REVIEW

File: src/features/sarcophagi/components/Rewrap.tsx
Comment (gh-23378) [inline src/features/sarcophagi/components/Rewrap.tsx:31]: 🤌

**Review by gh-23378 — APPROVED** [review]

**Comment by knoll3** [comment]
@DarksightKellar This is good to merge after merge conflicts have been addressed.

**Review by knoll3 — APPROVED** [review]

File: src/features/embalm/stepContent/steps/SelectArchaeologists.tsx
Comment (DarksightKellar) [inline src/features/embalm/stepContent/steps/SelectArchaeologists.tsx:63]: I realised when resolving merge conflicts that the thing this was solving no longer exists. This PR will bring it back (ie the extra Heading on the Archaeologists page), but we can revert this later.

**Pull request merged**
